File: pyg_parent/pyg_manager_web/src/main/webapp/js/controller/brandController.js
app.controller("brandController", function ($scope, $http, brandService,$controller) {
    $controller("baseController", {$scope: $scope});

    $scope.findAll=function () {
        brandService.findAll().success(function (res) {
            $scope.list=res;
        })
    };

    $scope.findPage=function (page, size) {
        brandService.findPage(page,size).success(
            function (response) {
                $scope.list = response.rows;
                $scope.paginationConf.totalItems = response.total;//更新总记录数
            }
        );
    };

    //新建
    $scope.entity = {};
    $scope.save=function () {
        brandService.save($scope.entity).success(function (res) {
            alert(res.message);
            if (res.success) {
                $scope.reloadList();
            }
        });
    };


    //修改
    $scope.findOne = function (id) {
        brandService.findOne(id).success(function (res) {
            $scope.entity = res;
        })
    };

    //删除
    $scope.delete = function () {
        if ($scope.ids.length == 0) {
            alert("至少选中一个")
        } else {
            brandService.delete($scope.ids).success(function (res) {
                alert(res.message);
                if (res.success) {
                    $scope.reloadList();
                    $scope.ids = [];
                }
            })
        }
    }
});

File: pyg_parent/pyg_manager_web/src/main/java/com/pinyougou/manager/controller/BrandController.java
package com.pinyougou.manager.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.pinyougou.pojo.TbBrand;
import com.pinyougou.sellergoods.service.BrandService;
import entity.PageResult;
import entity.Result;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
@RequestMapping("/brand")
public class BrandController {
    @Reference
    private BrandService brandService;

    @RequestMapping("/findAll")
    public List<TbBrand> findAll() {
        return brandService.findAll();
    }

    //分页
    @RequestMapping("/findPage/{page}/{size}")
    public PageResult<TbBrand> findPage(@PathVariable int page, @PathVariable int size) {
        return brandService.findPage(page, size);
    }

    //新建和修改
    @RequestMapping("/add")
    public Result save(@RequestBody TbBrand tbBrand) {
        try {
            brandService.save(tbBrand);
            return new Result(true, "保存成功");
        } catch (Exception e) {
            e.printStackTrace();
            return new Result(false, "保存失败");
        }
    }

    //回显
    @RequestMapping("/findOne/{id}")
    public TbBrand findOne(@PathVariable long id) {
        return brandService.findOne(id);
    }

    //删除
    @RequestMapping("/delete/{ids}")
    public Result delete(@PathVariable String ids) {
        try {
            brandService.delete(ids);
            return new Result(true, "删除成功");
        } catch (Exception e) {
            e.printStackTrace();
            return new Result(false, "删除失败");
        }
    }
}
